domain/src/main/java/com/teamagam/gimelgimel/domain/messages/MessagesTextSearcher.java

```diff
@@ -0,0 +1,8 @@
+package com.teamagam.gimelgimel.domain.messages;
```

Comment:
I'd pack everything related to the message-search under a new package named search in message package.

Reply:
done